templates/ovncontroller/bin/functions

```diff
@@ -57,34 +57,86 @@ function configure_external_ids {
     fi
 }
 
+# Returns the set difference between $1 and $2
+function set_difference {
+    echo "$(comm -23 <(sort <(echo $1 | xargs -n1)) <(sort <(echo $2 | xargs -n1)))"
+}
+
 # Configure bridge mappings and physical bridges
 function configure_physical_networks {
+    #local z=""
```

Comment:
That was a debug code and needs to be removed

Reply:
same timing :). will drop